code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -80,6 +86,12 @@
 				return
 			accesses -= SSid_access.get_region_access_list(list(region))
 			. = TRUE
+		if("passedName")
+			passed_name = params["passedName"]
```

Comment:
You can't trust user input like this, it could be literally anything.

Check if its text, and limit its length.

Reply:
it can be a number, it doesnt matter what it is, as long as its the same for all doors